### PR TITLE
[Sepolia] Fix withdrawal network for fee vaults

### PR DESCRIPTION
**Pull request by cody-wang-cb**
It turned out that the [genesis fee vaults](https://sepolia.basescan.org/address/0xC0D3C0d3c0d3c0d3C0D3c0d3C0D3c0d3c0D30011) didn't have the correct value for withdrawal network, it should be L2 (1) instead of L1 (0). Thus updating the script to have the correct value.

Will need to rerun this script and verify the new contracts.

Performed a dry run and the script runs successfully
```
Script ran successfully.

== Logs ==
  Sequencer Fee Vault Impl address: 0x7ce53CAa10ae0090ce28276086eFaeADc15F71Df
  L1 Fee Vault Impl address: 0x9A33D2cac3cef6dC2F3e61c20417e78326d4Bdb7
  Base Fee Vault Impl address: 0x8C39078aEa04dEE6C8e2487BCfdB2b00d094b56C

## Setting up 1 EVM.

==========================

Chain 84532

Estimated gas price: 3.000000508 gwei

Estimated total gas used for script: 2121217

Estimated amount required: 0.006363652077578236 ETH
```

Change log to be approved: https://docs.google.com/document/d/1VTGKFdAq3BtoDPQMuH4E47g0t0u9H_fi1eo1RJMFOn8/edit?usp=sharing

### CODE REVIEW

File: sepolia/2024-01-22-deploy-revshare/records/UpdateFeeVaultRecipient.s.sol/84532/run-1706556198.json
Comment (cody-wang-cb) [inline sepolia/2024-01-22-deploy-revshare/records/UpdateFeeVaultRecipient.s.sol/84532/run-1706556198.json:1]: Does it make sense to remove this record or we should keep this?

File: sepolia/2024-01-22-deploy-revshare/script/UpdateFeeVaultRecipient.s.sol [hunk 1]
```diff
@@ -33,19 +33,19 @@ contract UpdateFeeVaultRecipient is Script {
         SequencerFeeVault sfvNew = new SequencerFeeVault(
             recipient,
             sfvOld.MIN_WITHDRAWAL_AMOUNT(),
-            sfvOld.WITHDRAWAL_NETWORK()
```
Comment (cody-wang-cb) [inline sepolia/2024-01-22-deploy-revshare/script/UpdateFeeVaultRecipient.s.sol:36]: I think the fact that this is wrong means there's some wrong configuration in the genesis file? Is it somewhere in the OP repo where we set this

Comment (cody-wang-cb) [reply sepolia/2024-01-22-deploy-revshare/script/UpdateFeeVaultRecipient.s.sol:36]: found it https://github.com/base-org/contract-deployments/blob/main/sepolia/2023-09-26-deploy/deploy-config/deploy-config.json#L37

File: sepolia/2024-01-22-deploy-revshare/records/UpdateFeeVaultRecipient.s.sol/84532/run-1706556198.json
Comment (mdehoog) [reply sepolia/2024-01-22-deploy-revshare/records/UpdateFeeVaultRecipient.s.sol/84532/run-1706556198.json:1]: i think it makes sense to keep it as a record

**Review by mdehoog — APPROVED** [review]

**Review by anikaraghu — APPROVED** [review]

**Pull request merged**